sql/core/src/test/resources/sql-tests/inputs/subquery/scalar-subquery/scalar-subquery-predicate.sql

```diff
@@ -279,3 +279,14 @@ HAVING   max(t1b) <= (SELECT   max(t2b)
                       FROM     t2
                       WHERE    t2c = t1c
                       GROUP BY t2c);
+
+-- TC 03.01
+-- explain a simple uncorelated scalar subquery
+SET spark.sql.codegen.wholeStage = true;
```

Comment:
So, this is a workaround to avoid `SQLQueryTestSuite`'s `codegenConfigSets` combinations, right?

Reply:
@dongjoon-hyun Yes.. this is to bypass the codegenconfigsets combination.. Do you want to mke this test case to a new .sql  instead of modifying an existing one ?